src/components/LeaderBoardHeader/LeaderBoardHeader.module.scss

```diff
@@ -0,0 +1,38 @@
+.wrapper {
+  &__header {
+    font-style: normal;
+    font-weight: bold;
+    font-size: 34px;
+    line-height: 32px;
+    color: #1a1c1d;
+    text-align: center;
```

Comment:
Why you need & here?

Reply:
> Why you need & here?

I need only font-size and text-align for title in my header